Tests/TestingTests/TestSupport/TestingAdditions.swift

```diff
@@ -287,6 +293,10 @@ extension Test.ID.Selection {
   }
 }
 
+// TODO: Provide some convenience utilities for our own unit tests to streamline
```

Comment:
Might be worth doing this as API. A function on `Configuration` like so maybe?

```swift
public mutating func setTestFilter(toMatch selection: Test.ID.Selection)
```

Reply:
I did this!